olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -151,6 +151,22 @@ spec:
        * with the ability to select a specific subset of events from a particular
          producer.
 
+
+    ## Knative Kafka
+
+    Knative Kafka is a **[Technology Preview feature](https://access.redhat.com/support/offerings/techpreview)!**
+
+    Apache Kafka is a distributed streaming platform and Knative Kafka adds
+    the following components:
+
+    1. A Knative Eventing Source to consume messages from an Apache Kafka topic
+    1. A Knative Eventing Channel that is backed by Apache Kafka topics
```

Comment:
Maybe use unorderd lists as in the entire rest of the CSV.

Reply:
@markusthoemmes thanks! I've now converted the list